modules/igv/1.0/envs/format_regions.yaml

```diff
@@ -0,0 +1,38 @@
+name: test-generate_igv_batch
```

Comment:
What is the "acting part" here? Is it just to have python env? We can then see if there are other envs in lcr-modules that we can re-use here instead of adding new one 😄

Reply:
This is a leftover artifact from when I tried to use VCF files to specify regions to snapshot, I removed it since it's not necessary anymore